File: Java/src/selectors/nameSelectorExample.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class nameSelectorExample {

	public static void main(String[] args) {
		WebDriver browser = new ChromeDriver();
		
		browser.navigate().to("https://techstepacademy.com/training-ground");
//		elements
		WebElement textInputField1 = browser.findElement(By.name("Input 1"));
		WebElement textInputField2 = browser.findElement(By.name("Input 2"));
		WebElement button1 = browser.findElement(By.name("butn1"));
		WebElement button2 = browser.findElement(By.name("butn2"));
		WebElement button3 = browser.findElement(By.name("butn3"));
		WebElement button4 = browser.findElement(By.name("butn4"));
		
//		actions
		textInputField1.sendKeys("sample text...");
		button1.click();
		browser.switchTo( ).alert( ).dismiss();
		button2.click();
		browser.switchTo( ).alert( ).dismiss();
		
		textInputField2.sendKeys("sample text...");
		button3.click();
		browser.switchTo( ).alert( ).dismiss();
		button4.click();
		browser.switchTo( ).alert( ).dismiss();
	}
}

File: C#/selectors/NameSelectorExample.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System.Threading;
class CssSelectorExample
{
    static void Main()
    {
        IWebDriver browser = new ChromeDriver();
        IAlert alert;

        browser.Navigate().GoToUrl("https://techstepacademy.com/training-ground");

        // elements
        IWebElement textInputField1 = browser.FindElement(By.Name("Input 1"));
        IWebElement textInputField2 = browser.FindElement(By.Name("Input 2"));
        IWebElement button1 = browser.FindElement(By.Name("butn1"));
        IWebElement button2 = browser.FindElement(By.Name("butn2"));
        IWebElement button3 = browser.FindElement(By.Name("butn3"));
        IWebElement button4 = browser.FindElement(By.Name("butn4"));

        // actions
        textInputField1.SendKeys("sample text...");

        button1.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        button2.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        textInputField2.SendKeys("sample text...");

        button3.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        button4.Click();
        alert = browser.SwitchTo().Alert();
        alert.Accept();

        Thread.Sleep(3000);

        browser.Quit();
    }
}